README.rst

```diff
@@ -0,0 +1,93 @@
+image here
+image target here
+
+image here
+image target here
+
+
+PDFmorph
+========================================================================
+# FIXME
+add description, sample below
+Code package for morphin' atomic pair distribution functions.
+
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3.7 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* setuptools         - tools for installing and setting up Python packages
+* Sphinx             - documentation generator for reStructuredText files
+* sphinx_rtd_theme   - user-friendly sphinx theme
+* Doctr              - deploys docs from Travis CI to GitHub pages
```

Comment:
Usually the requirements are for running the software, not for deploying the docs. I'd move the docs related things (and remove doctr outright) into their own subsection if we want them.

Reply:
see #21 for fix to requirements

Reply:
I removed the requirements in that PR